app/views/users/index.html.erb

```diff
@@ -1 +1,17 @@
-<h1>This is the home page</h1>
\ No newline at end of file
+<h1>List of all users</h1>
+
+<table>
+  <tr>
+    <th><b>Photo</b></th>
+    <th><b>Username</b></th>
+    <th><b>No of posts</b></th>
+  </tr>
+  <% @users.each do |user| %>
+    <tr>
+      <td><%= user.photo %></td>
```

Comment:
- [ ] this should be an image so it can look like the wireframe
![Screenshot from 2023-06-14 22-58-26](https://github.com/Cabraham1/Blog/assets/66097371/a7d0d25f-98e7-46d3-9356-ba112cb96846)